runtime/manual/node/migrate.md

```diff
@@ -142,127 +149,217 @@ app.listen(3000, () => {
 ```
 
 If you run it with `deno run server.js`, it would prompt you for a number of
-permissions required to execute the code and its dependencies. These prompts can
-show you what runtime permission flags need to be passed in to grant the access
-you need. Running the code above with the necessary permissions provided would
-look like this:
+permissions required to execute the code and its dependencies. DENO SECURE BY
```

Comment:
```suggestion
permissions required to execute the code and its dependencies. DENO IS SECURE BY
```

Reply:
Oops I think this was more of a placeholder that we forgot to fill out 😅